src/ChromeDevtoolsProtocol/Instance/Launcher.php

```diff
@@ -113,6 +114,13 @@ public function launch(ContextInterface $ctx, ...$args): ProcessInstance
 				throw new RuntimeException(sprintf("Executable [%s] not found.", static::DEFAULT_LINUX_EXECUTABLE));
 			}
 
+		} else if (stristr(PHP_OS, 'WIN')) {
```

Comment:
- Please use `"` for strings.
- `stristr("Darwin", "WIN") === 3` => breaks OS X. `PHP_OS` strings must _start_ with `Win`, so it should be `stripos(PHP_OS, "Win") === 0`

Reply:
Good catch, copied detection where OS X case would be above Windows.